apps/nextjs-website/src/components/atoms/CookieBannerScript/CookieBannerScript.tsx

```diff
@@ -0,0 +1,34 @@
+'use client';
+import { isProduction } from '@/config';
+
+function makeCookieScript(dataDomainScript?: string) {
+  return `
+    <script
```

Comment:
Just a question: wasn't it possible to use the [`<Script>` component of Next](https://nextjs.org/docs/pages/api-reference/components/script)?